ansible/roles/docker_py/tasks/main.yml

```diff
@@ -20,16 +19,21 @@
     update_cache=yes
 
 - name: installs docker-engine package
-  apt: name=docker-engine
+  apt: name=docker-engine{{ docker_apt_ver }}
 
 - name: enables and starts docker service
   service: >
     name=docker
     enabled=yes
     state=started
 
+# 1. ansible 2.2.2 declares 1.7.0+ as a requirement, but it's really 1.8.0+
+#    https://github.com/ansible/ansible-modules-core/issues/5515
+#    https://github.com/ansible/ansible/issues/20776
+# 2. 2.0 is probably unsupported due to renaming of `docker.Client` to `docker.APIClient`
+#    https://docker-py.readthedocs.io/en/stable/change-log.html
 - name: Install docker-py via pip in virtualenv
   pip: >
     name="docker-py"
-    version="1.7.0"
-    virtualenv="{{ ansible_env.HOME }}/venv"
+    version="1.10.6"
```

Comment:
I remembered I had issues with any version greater that `1.7.0` perhaps this is fixed now.
See also: https://github.com/TheTorProject/ooni-sysadmin/commit/139cfab43a931db1aa39a46c7cf7cb7abedc8439